src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppElasticBeanstalkLinux.recipe

```diff
@@ -302,6 +302,34 @@
             "AdvancedSetting": true,
             "Updatable": true
         },
+        {
+            "Id": "LoadBalancerScheme",
+            "Name": "Load Balancer Scheme",
+            "Category": "LoadBalancer",
+            "Description": "Specify \"Internal\" if your application serves requests only from connected VPCs. Public load balancers serve requests from the Internet.",
```

Comment:
nit: add quotes around "public" and make both "Internal" and "Public" lowercase to indicate what the actual values are.

Reply:
Will do, waiting for other reviewer's feedback before rework.

Reply:
Added quotes around "Public" via https://github.com/aws/aws-dotnet-deploy/compare/728ba2390e178d1ed44c2a832458ddcefee621e9..69845e4684d83f12f2ba998f5abb889dbbc86897

I left them capitalized since that's what we're showing in the UI (even though they're lowercase in CDK)
![image](https://github.com/aws/aws-dotnet-deploy/assets/1170880/3f4b9317-1cc9-426e-86b8-777a8277c3ed)